File: app/src/main/java/com/liusbl/listbinderssample/MainActivity.java
package com.liusbl.listbinderssample;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.liusbl.listbinderssample.multi.MultiActivity;
import com.liusbl.listbinderssample.single.SingleActivity;

public final class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        findViewById(R.id.singleViewTypeExampleButton).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(MainActivity.this, SingleActivity.class);
                startActivity(intent);
            }
        });

        findViewById(R.id.multiViewTypeExampleButton).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(MainActivity.this, MultiActivity.class);
                startActivity(intent);
            }
        });
    }
}

File: app/src/main/java/com/liusbl/listbinderssample/MainActivity.kt
package com.liusbl.listbinderssample

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.liusbl.listbinderssample.multi.MultiActivity
import com.liusbl.listbinderssample.single.SingleActivity
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        singleViewTypeExampleButton.setOnClickListener {
            val intent = SingleActivity.createIntent(this@MainActivity)
            startActivity(intent)
        }
        multiViewTypeExampleButton.setOnClickListener {
            val intent = MultiActivity.createIntent(this@MainActivity)
            startActivity(intent)
        }
    }
}